Formula/kibana.rb

```diff
@@ -49,6 +49,7 @@ def install
     prefix.install Dir["build/kibana-#{version}-#{platform.sub("x64", "x86_64")}/{bin,config,node_modules,optimize,package.json,src,webpackShims}"]
 
     inreplace "#{bin}/kibana", %r{/node/bin/node}, "/libexec/node/bin/node"
+    inreplace "#{bin}/kibana-plugin", %r{/node/bin/node}, "/libexec/node/bin/node"
```

Comment:
Could maybe do a `Dir["#{bin}/*"].each` to cover any future libraries that were added in here? I have no strong feelings either way.